article/app/controllers/LiveBlogController.scala

```diff
@@ -35,14 +35,20 @@ class LiveBlogController(contentApiClient: ContentApiClient, val controllerCompo
       }
     }
 
-  def renderHeadline(path: String): Action[AnyContent] = {
-    Action.async { implicit request =>
-      mapModel(path, ArticleBlocks) { page =>
-        Future.successful(Cached(page)(RevalidatableResult.Ok(page.metadata.webTitle)))
-      }
+  def renderHeadline(path: String): Action[AnyContent] = Action.async { implicit request =>
```

Comment:
I've removed this in a subsequent commit and changed the routes file to point at the same method in ArticleController